toolchain/driver/driver_env.h

```diff
@@ -18,15 +20,19 @@ struct DriverEnv {
   // Helper to locate the toolchain installation's files.
   const InstallPaths* installation;
 
+  // Standard input; stdin. May be null, to prevent accidental use.
+  FILE* input_stream;
   // Standard output; stdout.
-  llvm::raw_pwrite_stream& output_stream;
+  llvm::raw_pwrite_stream* output_stream;
   // Error output; stderr.
-  llvm::raw_pwrite_stream& error_stream;
+  llvm::raw_pwrite_stream* error_stream;
```

Comment:
How come these became pointers?

Reply:
Style:

> When storing an object's address as a non-owned member, prefer storing a pointer.

https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/cpp_style_guide.md#syntax-and-formatting

Note this is covered by #4812